src/plugins/spawnables/spawnables.game.ts

```diff
@@ -139,37 +116,43 @@ export const SpawnablesModule = defineModule<GameContext, SpawnablesModuleState>
     });
     blackMirrorMaterial.addRef();
 
-    registerPrefab(ctx, {
-      name: "small-ball",
-      type: PrefabType.Object,
-      create: (ctx, { kinematic }) => {
-        return createBall(ctx, module, physics, 0.25, emissiveMaterial, ballAudioData, kinematic);
-      },
-    });
-
-    registerPrefab(ctx, {
-      name: "mirror-ball",
-      type: PrefabType.Object,
-      create: (ctx, { kinematic }) => {
-        return createBall(ctx, module, physics, 1, mirrorMaterial, ballAudioData, kinematic);
-      },
-    });
-
-    registerPrefab(ctx, {
-      name: "black-mirror-ball",
-      type: PrefabType.Object,
-      create: (ctx, { kinematic }) => {
-        return createBall(ctx, module, physics, 1, blackMirrorMaterial, ballAudioData, kinematic);
-      },
-    });
-
-    registerPrefab(ctx, {
-      name: "emissive-ball",
-      type: PrefabType.Object,
-      create: (ctx, { kinematic }) => {
-        return createBall(ctx, module, physics, 2, emissiveMaterial, ballAudioData, kinematic);
-      },
-    });
+    module.replicators = {
```

Comment:
We should come up with a better way of registering replicators on load/enter world for our own internal code.